Bottom: 8c81211cf6460c594428b52aa1831c82fb9ccf89
Top:    a38f4698dc5dbb97b7ccef09ef3a65cb2cc4070d
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 16:02:08 +0800

Refresh of vhost-let-vhost_signal-returns

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index de8b2b5..faa0dd7 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -165,7 +165,7 @@ void vhost_add_used_and_signal(struct vhost_dev *, struct vhost_virtqueue *,
 			       unsigned int id, int len);
 void vhost_add_used_and_signal_n(struct vhost_dev *, struct vhost_virtqueue *,
 			       struct vring_used_elem *heads, unsigned count);
-bool vhost_signal(struct vhost_dev *, struct vhost_virtqueue *);
+void vhost_signal(struct vhost_dev *, struct vhost_virtqueue *);
 void vhost_disable_notify(struct vhost_dev *, struct vhost_virtqueue *);
 bool vhost_enable_notify(struct vhost_dev *, struct vhost_virtqueue *);
